src/components/themed/SideMenu.tsx

```diff
@@ -62,6 +62,8 @@ export function SideMenu(props: DrawerContentComponentProps) {
   // Maintain the list of usernames:
   const localUsers = useWatch(context, 'localUsers')
   const watchedUsername = useWatch(account, 'username')
+  const displayUsername =
+    watchedUsername == null ? sprintf(lstrings.guest_account_id_1s, account.rootLoginId.slice(account.rootLoginId.length - 3)) : watchedUsername
```

Comment:
Should this be in a common helper function, in case we ever change the logic?

Reply:
I considered it, but we'd need to wire it into loginui too somehow. A getter on
the account and userinfo seems most appropriate?